src/elements/ts-profile/ts-options-menu.html

```diff
@@ -92,8 +92,12 @@
         },
 
         create: function () {
-            this.set('newinfo', {});
-            this.set('selected', 2);
+            // To restore the original behavior where this opens the
+            // "Create Account" form, uncomment the next two lines:
+            // this.set('newinfo', {});
+            // this.set('selected', 2);
+            var createaccounturl = App.configurator.getUserSetting('createaccounturl');
+            require("electron").shell.openExternal(createaccounturl);
```

Comment:
We should double check that this works on all platforms.

Reply:
I confirmed it works on Windows and Linux.  I'll check with John to see if he can test it on a Mac.